hypervisor/src/mshv/mod.rs

```diff
@@ -929,6 +932,22 @@ impl vm::Vm for MshvVm {
             )
             .map_err(|e| vm::HypervisorVmError::GetDirtyLog(e.into()))
     }
+    ///
+    /// Enable dirty page tracking
+    ///
+    fn enable_dirty_page_tracking(&self) -> vm::Result<()> {
+        self.fd
+            .enable_dirty_page_tracking()
+            .map_err(|e| vm::HypervisorVmError::EnableDirtyPageTracking(e.into()))
+    }
+    ///
+    /// Disable dirty page tracking
+    ///
+    fn disable_dirty_page_tracking(&self) -> vm::Result<()> {
+        self.fd
+            .disable_dirty_page_tracking()
+            .map_err(|e| vm::HypervisorVmError::DisableDirtyPageTracking(e.into()))
+    }
```

Comment:
Similarly, `self.fd.disable_dirty_page_tracking()` is better to be called directly from `stop_dirty_log()`. In this way, you won't need to change the logic from the `vmm` crate at all.